README.md

```diff
@@ -29,6 +29,10 @@ remoteagent1-dcos112s.example.com
 [agents_public]
 publicagent1-dcos112s.example.com
 
+[win_agents]
```

Comment:
I think we should state the non default transport here

ansible_connection: winrm
ansible_winrm_transport: basic